pvlib/iotools/__init__.py

```diff
@@ -5,8 +5,6 @@
 from pvlib.iotools.surfrad import read_surfrad  # noqa: F401
 from pvlib.iotools.midc import read_midc  # noqa: F401
 from pvlib.iotools.midc import read_midc_raw_data_from_nrel  # noqa: F401
-from pvlib.iotools.ecmwf_macc import read_ecmwf_macc  # noqa: F401
-from pvlib.iotools.ecmwf_macc import get_ecmwf_macc  # noqa: F401
```

Comment:
The entries in `docs/sphinx/source/reference/iotools.rst` need to be removed too